src/site/content/en/blog/prefers-color-scheme/index.md

```diff
@@ -0,0 +1,786 @@
+Note how I don’t include any value, but purely check if the media query alone matches.
+Actually, I have silently used this pattern already above in the
+[loading strategy](#loading-strategy) code snippet.
+
+```js
+if (window.matchMedia('(prefers-color-scheme)').media !== 'not all') {
+  console.log('🎉 Dark mode is supported');
+}
+```
+
+At the time of writing, `prefers-color-scheme` is supported on both desktop and mobile (where available)
+by Chrome and Edge as of version 76, Firefox as of version 67,
+and Safari as of version 12.1 on macOS and as of version 13 on iOS.
+For all other browsers, you can check the [Can I use support tables](https://caniuse.com/#feat=prefers-color-scheme).
+
+{% Aside 'note' %}
+  There is a custom element [`<dark-mode-toggle>`](https://github.com/GoogleChromeLabs/dark-mode-toggle)
+  available that adds dark mode support to older browsers.
+  I write about it [further down in this article](#the-lessdark-mode-togglegreater-custom-element).
+{% endAside%}
+
+### Reacting on dark mode changes
+
+Like any other media query change, dark mode changes can be subscribed to via JavaScript.
+You can use this to, for example, dynamically change the
+[favicon](https://developers.google.com/web/fundamentals/design-and-ux/browser-customization/#provide_great_icons_tiles)
+of a page or change the
+[`<meta name="theme-color">`](https://developers.google.com/web/fundamentals/design-and-ux/browser-customization/#meta_theme_color_for_chrome_and_opera)
+that determines the color of the URL bar in Chrome.
+The [full example](#full-example) above shows this in action,
+in order to see the theme color and favicon changes, open the
+[demo in a separate tab](https://dark-mode-baseline.glitch.me/).
+
+```js
+  const darkModeMediaQuery = window.matchMedia('(prefers-color-scheme: dark)');
+  darkModeMediaQuery.addListener((e) => {
+    const darkModeOn = e.matches;
+    console.log(`Dark mode is ${darkModeOn ? '🌒 on' : '☀️ off'}.`);
+  });
+```
+
+## Dark mode best practices
+
+### Avoid pure white
+
+A small detail you may have noticed is that I don’t use pure white.
+Instead, to prevent glowing and bleeding against the surrounding dark content,
+I choose a slightly darker white. Something like `rgb(250, 250, 250)` works well.
+
+### Re-colorize and darken photographic images
+
+If you compare the two screenshots below, you will notice that not only the core theme has changed
+from *dark-on-light* to *light-on-dark*, but that also the hero image looks slightly different.
+My [user research](https://medium.com/dev-channel/re-colorization-for-dark-mode-19e2e17b584b)
+has shown that the majority of the surveyed people
+prefer slightly less vibrant and brilliant images when dark mode is active.
+I refer to this as *re-colorization*.
+
+<figure>
+  <div style="width: 100%">
+    <div style="display: inline-block;
+      width: 45%;
+      height: 100px;
+      background-image: url(dark.png);
```

Comment:
I've spent maybe 30 minutes trying to track down these images. I can see them showing up when I preview the site but there are separate files in this directory also called dark.png and light.png and they're different from these images.

![image](https://user-images.githubusercontent.com/1066253/60221163-77a89d00-982d-11e9-97bb-e05f55bc51fa.png)

I'm going to change this so it uses [our columns component](https://web.dev/handbook/web-dev-components/#columns) instead of inline styles. But I think you'll need to swap these images back in.

Reply:
OH! I just looked at the styles. It's using CSS to crop one of the existing images.

Instead I'd suggest using actual screenshots. I'll put the `w-columns` code in place and you can swap out the images.

Reply:
I'm now using the `w-columns` code universally.
https://github.com/GoogleChrome/web.dev/pull/1021/commits/a32246b9621447b0a607db076c8c3c0b5a3c9423 and https://github.com/GoogleChrome/web.dev/pull/1021/commits/164566e5545a30ad15a83a267ad7350d21fd266e.